### PR TITLE
PMUX-1324 clisam cli temp master

### PR DESCRIPTION
**Pull request by JoeMulka**
Hi All,
This change adds CLI support for the somatic assay model.  The spec for this [can be found here](https://docs.google.com/document/d/1UjhTQh_w8iuD3OZ1AJs3WAVuUlx9TKK62bZzcrx3fr0/edit?pli=1)
Additional automated testing is forthcoming
Please feel free to make change requests and leave comments, and also to DM me directly with any questions or requests

### CODE REVIEW

File: src/python/test/test_extract_somatic.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+#
+# Select test suite
+#
+dataset = "single_assay"
+
+if dataset == "single_assay":
+    # Single assay
+    test_project = "PMUX-1324-SCIPROD-CLISAM"
+    test_record = "{}:/test_keegan_202306231200".format(test_project)
```
Comment (keegankelsey) [inline src/python/test/test_extract_somatic.py:52]: Should be updated with `test_single_assay_202306231200` @JoeMulka

File: src/python/dxpy/dx_extract_utils/somatic_filter_payload.py [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+            "compound":[
+                {
+                    <contents of location compound>
+                }
+            ]
+        }
+    }
+    """
+    pheno_filter = {
+        "pheno_filters": {"name": name, "id": id, "logic": "and", "compound": []}
```
Comment (durae) [inline src/python/dxpy/dx_extract_utils/somatic_filter_payload.py:144]: Tomorrow (6/27) we will push to vizserver the update, which will allow to send the filters as `assay_filters` and not `pheno_filters`. However, the change should land on production only on 7/11. Please update the code dependent on the release plan for this feature.
cc: @keegankelsey

**Review by kpjensen — CHANGES_REQUESTED** [review]
Please revert the 8k lines of linter changes in `dx.py` so that this PR only includes the relevant changes for this feature.

**Review by kpjensen — APPROVED** [review]

File: src/python/dxpy/dx_extract_utils/somatic_filter_payload.py (hunk 2 above)
Comment (durae) [reply src/python/dxpy/dx_extract_utils/somatic_filter_payload.py:153]: @JoeMulka can this be changed to use assay_filters properly? The vizserver support is planned to go on production next Tuesday 7/11 so definitely before CLISAM release.

**Pull request merged**